lib/stupidedi/versions/functional_groups/004010/element_defs.rb

```diff
@@ -64,7 +64,7 @@ module ElementDefs
           E60   = t:: R.new(:E60  , "Freight Rate"                         , 1, 9)
           E61   = t::AN.new(:E61  , "Free-Form Message"                    , 1, 30)
           E65   = t:: R.new(:E65  , "Height"                               , 1, 8)
-          E66   = t::ID.new(:E66  , "Identification Code Qualifier"        , 2, 3,
+          E66   = t::ID.new(:E66  , "Identification Code Qualifier"        , 1, 3,
```

Comment:
Really not sure about this one, but the specs we got for our case state that E66 should be length 1-2, not 2-3.